src/System.Management.Automation/engine/PSVersionInfo.cs

```diff
@@ -413,7 +413,7 @@ public SemanticVersion(int major, int minor, int patch, string label)
         /// <exception cref="PSArgumentException">
         /// If <paramref name="major"/>, <paramref name="minor"/>, or <paramref name="patch"/> is less than 0.
         /// </exception>
-        public SemanticVersion(int major, int minor, int patch)
+        public SemanticVersion(int major, int minor = 0, int patch = 0)
```

Comment:
We should not use default arguments in public apis. The guidance is to provide multiple overloads for languages that may not support default arguments.